### DISCUSSION TITLE
`<TagGroup>` component has no FieldError

### DISCUSSION THREAD

**Discussion by aromko** [adobe/react-spectrum#5473] [General]
Hi. Is there a reason why the TagGroup component has no FieldErrorContext? Will it be added in the future?

Thanks in advance.

**Comment by snowystinger** [comment]
TagGroup isn't something which can be submitted to the server, it has no native input backing it. We may eventually have a TagField or something along those lines, in which case that would have FieldError, but TagGroup is just a static set of tags, it includes no control for adding them/creating them. As a result, it's probably best to do the validation/errors before a tag is even added to a tag group.

If you're using TagGroup selection, I suggest moving to a Checkbox Group instead. If we have a TagField eventually, I suspect it will not have selection, as adding a Tag to the TagGroup and Selection would be at odds with each other.
You can style your Checkboxes/CheckboxGroup to look like Tags.

**Reply by aromko** [reply]
Ok thanks for the informations. So the issue can be closed. Have a nice day.
